libyul/AsmAnalysis.cpp

```diff
@@ -766,3 +768,35 @@ bool AsmAnalyzer::validateInstructions(FunctionCall const& _functionCall)
 {
 	return validateInstructions(_functionCall.functionName.name.str(), nativeLocationOf(_functionCall.functionName));
 }
+
+bool AsmAnalyzer::validateObjectStructure(langutil::SourceLocation _astRootLocation)
```

Comment:
Maybe we could drop the return value here? You're not checking if anyway.

More importantly, it's the opposite of what I'd expect - `true` for invalid. Are you maybe following what our weird `validateInstruction()` was doing? I wanted to change that actually, because that was confusing as hell. Let's not let it spread :)

Reply:
I just wanted to be consisted with this convention "Some errors -> return true". :) But yeah let's skip return value.

Reply:
Yeah, it's good to be consistent, just not sure we actually have that convention. I only recall it in that one function :)